benchmark_runner/main/main.py

```diff
@@ -159,10 +159,10 @@ def upgrade_ocp_bare_metal(step: str):
     bare_metal_operations.connect_to_provisioner()
     oc = bare_metal_operations.oc_login()
     if step == 'run_bare_metal_ocp_upgrade':
-        bare_metal_operations.run_ocp_upgrade(oc)
-        # The LSO/ODF upgrade must be run manually after the OCP upgrade for the channel version; it won't upgrade automatically.
-        bare_metal_operations.install_ocp_resources(resources=['lso'], upgrade_version=lso_version)
-        bare_metal_operations.install_ocp_resources(resources=['odf'], upgrade_version=odf_version)
+        if not bare_metal_operations.run_ocp_upgrade(oc) == 'Cluster is already upgraded':
```

Comment:
Is there anything you need to do if the status is anything different (e. g. if either the upgrade fails or the upgrade does something, so it's not previously upgraded)?

Reply:
Add dedicate method is_ocp_already_upgraded